gallery/src/ha-gallery.js

```diff
@@ -31,10 +31,21 @@ class HaGallery extends PolymerElement {
         visibility: hidden;
       }
 
-      paper-card {
+      .pickers {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: center;
+        align-items: start;
```

Comment:
default

Reply:
![image](https://user-images.githubusercontent.com/1444314/43066077-f66be252-8e63-11e8-87ee-dbdf3ed6eb8e.png)

Reply:
(the default is 'normal')